### ISSUE TITLE
🚸 Autocomplete: improve ux of singleselect

### ISSUE THREAD

**Issue by oddvernes** [equinor/design-system#3070]
[Discussion on slack](https://equinor.slack.com/archives/CJT20H1B9/p1695366581731959). There is no visual distinction between selected value and search string in the input field, which can lead to people thinking they have selected a value when the real selected value is something completely different. Vegar Vikan suggested the dropdown is always open with a "no match found" text when the text is search input. I then suggested the text should revert to selected value on blur as well
